sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -1,39 +1,40 @@
 # Azure Service Bus client library for Java
 
 Microsoft Azure Service Bus is a fully managed enterprise integration message broker. Service Bus can decouple
-applications and services. Service Bus offers a reliable and secure platform for asynchronous transfer of data
-and state. Data is transferred between different applications and services using messages. If you would like to know
-more about Azure Service Bus, you may wish to review: [What is Service Bus](https://docs.microsoft.com/en-us/azure/service-bus-messaging)?
+applications and services. Service Bus offers a reliable and secure platform for asynchronous transfer of data and
+state. Data is transferred between different applications and services using messages. If you would like to know more
+about Azure Service Bus, you may wish to review: [What is Service Bus][product_docs]?
```

Comment:
Is the `?` meant to be part of the link name?

Reply:
Good catch.